### PR TITLE
[COMMON] Add and fix esoc links, add CamX postprocessing manifest

### PR DESCRIPTION
**Pull request by kholk**
The MPSS readwrite link was not properly updated.
Also, add the 'tn' links, as the new platforms need it.

### CODE REVIEW

File: hardware/tftp/Android.mk [hunk 1]
```diff
@@ -54,6 +57,13 @@ SONY_SYMLINKS += \
         /data/vendor/tombstones/rfs/slpi:$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/slpi/ramdumps \
     )
 
+ifeq ($(TARGET_USES_ESOC),true)
+SONY_SYMLINKS += \
+    $(foreach tgtsoc,$(target_soc), \
+        /data/vendor/tombstones/rfs/tn:$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/tn/ramdumps \
+    )
+endif
```
Comment (MarijnS95) [inline hardware/tftp/Android.mk:65]: At this point we might as well wrap these in a:

```mk
SONY_SYMLINKS += \
    $(foreach tgtsoc,$(target_soc), \
        $(foreach prefix,$(target_prefixes), \
            /data/vendor/tombstones/rfs/$(prefix):$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/$(prefix)/ramdumps
        ) \
    )
```
And accept that the one for `adsp` and `mpss` have a different "from" path. Can't hurt to have those symlinks /methinks, the pattern is generic enough and already repeating for `cdsp`/`slpi`/`tn`.

**Review by MarijnS95 — APPROVED** [review]

**Comment by kholk** [comment]
I wanted to make that a new PR, but I've inadvertently pushed in this one.
Aadds the QTI postprocessing interface manifest to the EDO platform.

File: hardware/tftp/Android.mk (hunk 1 above)
Comment (kholk) [reply hardware/tftp/Android.mk:65]: Maybe. If you want, do a followup after this PR gets in.

File: hardware/tftp/Android.mk [hunk 2]
```diff
@@ -41,7 +44,7 @@ SONY_SYMLINKS := \
 SONY_SYMLINKS += \
     $(foreach tgtsoc,$(target_soc), \
         $(foreach prefix,$(target_prefixes), \
-            /mnt/vendor/persist/rfs/msm/$(prefix):$(TARGET_COPY_OUT_VENDOR)/rfs/msm/$(prefix)/readwrite \
+            /mnt/vendor/persist/rfs/$(tgtsoc)/$(prefix):$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/$(prefix)/readwrite \
```
[the remark this answers is not in the document]
Comment (kholk) [reply hardware/tftp/Android.mk:47]: Right, man? :P

**Comment by MarijnS95** [comment]
> Aadds the QTI postprocessing interface manifest to the EDO platform.

Who's hosting this, and why is it not a vintf fragment?

File: hardware/tftp/Android.mk (hunk 1 above)
Comment (MarijnS95) [reply hardware/tftp/Android.mk:65]: Now would be nice though...

**Comment by jerpelea** [comment]
@kholk partial cherrypick 

please rebase

File: common-binds.mk [hunk 3]
```diff
@@ -28,6 +28,21 @@
...
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib/libadsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib/libcdsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib/libmdsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib/libsdsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib64/libOpenCL_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib64/libQSEEComAPI_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib64/libadsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib64/libcdsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib64/libmdsprpc_system.so \
+    $(COMMON_PATH)/misc/emptyfile.zip:$(TARGET_COPY_OUT_PRODUCT)/lib64/libsdsprpc_system.so
```
Comment (MarijnS95) [inline common-binds.mk:44]: Force-push lost my comment. Please use `emptyfile` for non-zip things like shared objects, and keep `zip` reserved for things that need to appear like a Java thingy like an `apk` or `jar`.

**Review by MarijnS95 — CHANGES_REQUESTED** [review]

**Comment by jerpelea** [comment]
cherry-picked

**Pull request closed without merging**